service/dap/server_test.go

```diff
@@ -1997,7 +2078,7 @@ func TestLaunchRequestWithStackTraceDepth(t *testing.T) {
 				execute: func() {
 					client.StackTraceRequest(1, 0, 0)
 					stResp = client.ExpectStackTraceResponse(t)
-					checkStackFrames(t, stResp, "main.Increment", 8, 1000, 2 /*returned*/, 2 /*available*/)
+					checkStackFrames(t, stResp, "main.Increment", 8, 1000, 1 /*returned*/, 2 /*available*/)
```

Comment:
Changed back. The number of levels to be loaded should be s.args.stackTraceDepth + 1, not s.args.stackTraceDepth.

Reply:
Ok I'm reconsidering changing it back.

Do we think setting s.args.stackTraceDepth to 1 should mean we get 1 frame back, or 2 frames back? I thought that getting 1 frame back would give the user the most control over how many frames they load, since they cannot set the stackTraceDepth to 0.

Reply:
I am pretty sure the math was based on what the legacy adapter had. And in that case we should have accepted 0 to result in 1 frame. But you are right, its likely more intuitive to just have depth be the number of frames.